pset-functions.js

```diff
@@ -207,6 +271,20 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 5:')
 
 // Add your code below this line
+function doubleLetters(word) {
+    result = false
+    letters = Array.from(word)
+    for (let i = 0; i < letters.length; i++) {
+        if (i + 1 < letters.length) {
+            letter = letters[i]
+            if (letter == letters[i + 1]) {
+                result = true
+            }
+        }
+    }
+    console.log(result)
+}
+doubleLetters('loop')
```

Comment:
@laraann88 Good work here coming up with this solution. I see you are utilizing that array method .from which creates a shallow copy of the array. Good work here one suggestion, try to avoid nested conditional statements in your code. See if you can figure out an alternative solution to resolve the nested conditions.

Take a look at a solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L291)